compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -480,7 +480,81 @@ TR::Register *
 OMR::ARM64::TreeEvaluator::tableEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 	{
 	// TODO:ARM64: Enable TR::TreeEvaluator::tableEvaluator in compiler/aarch64/codegen/TreeEvaluatorTable.hpp when Implemented.
-	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
+   int32_t       total = node->getNumChildren();
+   int32_t       numBranchTableEntries = total - 2;
+   TR::Register   *selectorReg           = cg->evaluate(node->getFirstChild());
+   int32_t       i;
+   TR::Node       *secondChild = node->getSecondChild();
```

Comment:
Please choose a better name for this child to make the code more readable.  Perhaps the `defaultChild`?